packages/opencensus-propagation-b3/AUTHORS

```diff
@@ -0,0 +1 @@
+Google Inc.
```

Comment:
Should be `Google LLC.`.  Should CESAR be included here?

Reply:
Whoops, I see this has already been addressed. I was looking at a subset of the commits.